portability-transfer/src/main/java/org/datatransferproject/transfer/JobPollingService.java

```diff
@@ -153,11 +153,6 @@ private boolean tryToClaimJob(UUID jobId, WorkerKeyPair keyPair) {
     // Lookup the job so we can append to its existing properties.
     PortabilityJob existingJob = store.findJob(jobId);
     monitor.debug(() -> format("JobPollingService: tryToClaimJob: jobId: %s", existingJob));
-    // Verify no transfer worker key
```

Comment:
Since the state check is being done in the default impl of `claimjob` this should be safe because checking the state is still creds_avaialble is the same as checking if this job has the worker key information since a state change happens when that information is written

if this turns out to be an issue, anyone requiring this check can add it in as an additional check in claimjob in their impl of jobstore

 https://github.com/dtinit/data-transfer-project/blob/d228037bb8e661cd1ebe31c517c6721ed90652d6/portability-spi-cloud/src/main/java/org/datatransferproject/spi/cloud/storage/JobStoreWithValidator.java#L19-L23